rig/machine_control/machine_controller.py

```diff
@@ -132,6 +132,17 @@ def scp_data_length(self):
             self._scp_data_length = data.buffer_size
         return self._scp_data_length
 
+    @property
+    def scp_window_size(self):
+        """The maximum number of packets that can be sent to a SpiNNaker board
+        without receiving any acknowledgement packets.
+        """
+        # If not known, return the default
+        # TODO: Query the machine
+        if self._window_size is None:
+            return 5
```

Comment:
Default should be 1, should actually query the machine to determine what this value is.